ansible/templates/awx-operator.yaml.j2

```diff
@@ -10,3 +10,9 @@
 {% include 'service_account.yml.j2' %}
 
 {% include 'operator.yml.j2' %}
+
+{% include 'crd.yml.j2' %}
```

Comment:
This looks like a botched rebase. If you look at devel, this has been moved to the top of this file. The other CRDs should also be moved up before the other resources.

Reply:
I had done that intentionally thinking that the order mattered.  turns out the backup and restore CRD's need to exist prior to deploying the operator, so the order doesn't matter anyways where.  I'll put it back so it matches devel.

Reply:
Oh, I see now, there are duplicates